NtupleProducer/plugins/bRegressionProducer.cc

```diff
@@ -0,0 +1,570 @@
+            cout<<"Jet_energyRing_dR0_neut_Jet_e :"<<Jet_energyRing_dR0_neut_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR1_neut_Jet_e :"<<Jet_energyRing_dR1_neut_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR2_neut_Jet_e :"<<Jet_energyRing_dR2_neut_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR3_neut_Jet_e :"<<Jet_energyRing_dR3_neut_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR4_neut_Jet_e :"<<Jet_energyRing_dR4_neut_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR0_ch_Jet_e :"<<Jet_energyRing_dR0_ch_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR1_ch_Jet_e :"<<Jet_energyRing_dR1_ch_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR2_ch_Jet_e :"<<Jet_energyRing_dR2_ch_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR3_ch_Jet_e :"<<Jet_energyRing_dR3_ch_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR4_ch_Jet_e :"<<Jet_energyRing_dR4_ch_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR0_mu_Jet_e :"<<Jet_energyRing_dR0_mu_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR1_mu_Jet_e :"<<Jet_energyRing_dR1_mu_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR2_mu_Jet_e :"<<Jet_energyRing_dR2_mu_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR3_mu_Jet_e :"<<Jet_energyRing_dR3_mu_Jet_e <<endl;
+            cout<<"Jet_energyRing_dR4_mu_Jet_e :"<<Jet_energyRing_dR4_mu_Jet_e <<endl;
+            cout<<"Jet_numDaughters_pt03 :"<<Jet_numDaughters_pt03 <<endl;
+            cout<<"Jet_chHEF:"<<Jet_chHEF<<endl;
+            cout<<"Jet_chEmEF:"<<Jet_chEmEF<<endl;
+            cout<<"Jet_leptonPtRelInv:"<<Jet_leptonPtRelInv<<endl;
+            cout<<"isEle:"<<isEle<<endl;
+            cout<<"isMu:"<<isMu<<endl;
+            cout<<"isOther:"<<isOther<<endl;
+            cout<<"Jet_mass:"<<Jet_mass<<endl;
+            cout<<"Jet_withPtd:"<<Jet_withPtd<<endl;
+            cout << endl;
+
+
+        }
+
+        //        //..... gen jets info     [ DO WE NEED TO DO SOMETHING LIKE THAT?]                                                                              
+        //        int cflav = 0; //~correct flavour definition
+        //        if ( !evt.isRealData() ) {
+        //            int hflav = fjet.hadronFlavour();//4 if c, 5 if b, 0 if light jets
+        //            int pflav = fjet.partonFlavour();
+        //
+        //            if( hflav != 0 ) {
+        //                cflav = hflav;
+        //            } else { //not a heavy jet                                              
+        //                cflav = std::abs(pflav) == 4 || std::abs(pflav) == 5 ? 0 : pflav;
+        //            }
+        //            std::cout<< "cflav==" << cflav<<std::endl;
+        //            //if (cflav != 5) continue;//i want only bjets
+        //        }
+
+
+        SetNNVectorVar();
+        bRegNN = EvaluateNN();
+        NNvectorVar_.clear();
+
+        float y_mean= 1.0454729795455933;
+        float y_std = 0.3162831664085388;
```

Comment:
Maybe add a comment specifying where these numbers come from.

Reply:
The comment at the top of the file  does answer this I suppose

Reply:
I agree it helps, but if that other file is modified no one will ever know where these numbers came from (they do not appear in the AN).
And if you look for the configuration mentioned in the liked file, it is gone!

```
bfontana@lxplus737:/afs/cern.ch/work/b/bfontana  $ ll /afs/cern.ch/user/n/nchernya/public/100M_2018-03-01_job23_rawJetsJECtarget/config.json
ls: cannot access /afs/cern.ch/user/n/nchernya/public/100M_2018-03-01_job23_rawJetsJECtarget/config.json: No such file or directory
```

Reply:
now arguments in producer; values updated according to what is found in year-by-year json (in `data` directory)